extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/extractors/YoutubeCommentsInfoItemExtractor.java

```diff
@@ -248,6 +249,15 @@ public String getUploaderUrl() throws ParsingException {
         }
     }
 
+    @Override
+    public int getReplyCount() throws ParsingException {
+        try {
+            return JsonUtils.getNumber(json, "comment.commentRenderer.replyCount").intValue();
+        } catch (final Exception e) {
+            return UNKNOWN_REPLY_COUNT;
+        }
+    }
+
```

Comment:
@Stypox @AudricV Is there any reason, why these have a try & catch? Shouldn't we throw an exception to get informed about the problem?

Reply:
I think you are right, but we need handle error in UI.

Reply:
But if continue to think about it, if Youtube change API, we need adapt it as quick as possible but not hide it.
So, I think we can throw the error and in Newpipe client, we check reply page and count, if we have one of these, that means there have comment replies.

Reply:
That means we need change [this line](https://github.com/TeamNewPipe/NewPipeExtractor/blob/4b4c7e869a22d22fc7a52ad76cf2ed36c8667a17/extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/extractors/YoutubeCommentsInfoItemExtractor.java#L262) also.

Reply:
All non-critical errors thrown here, are added to an error list in the collectors. They are  displayed via a snackbar or notification in NewPipe automatically.
I was asking @Stypox and @AudricV regarding the other code, because some of these thrown errors might just mean, that the info is not present due to some settings by the content creator.
The comment count however, should always be available and thus exceptions thrown.

Reply:
It's done https://github.com/TeamNewPipe/NewPipeExtractor/pull/936/commits/c4f6a69dbcbdad86207a25b1cb4e6ba8a4558237